statsmodels/regression/tests/test_quantile_regression.py

```diff
@@ -1,9 +1,9 @@
 import scipy.stats
 import numpy as np
-import statsmodels.api as sm
 from numpy.testing import assert_allclose, assert_equal, assert_almost_equal
-from patsy import dmatrices  # pylint: disable=E0611
+import statsmodels.api as sm
 from statsmodels.regression.quantile_regression import QuantReg
+from patsy import dmatrices  # pylint: disable=E0611
```

Comment:
Ideal order is

Std library

External packages

statsmodels


and within each block imports before from.

import a
import b
from c import d
from e import f